packages/esm-patient-chart-app/src/visit/visits-widget/visit-detail-overview.component.tsx

```diff
@@ -42,6 +49,37 @@ function VisitDetailOverviewComponent({ patientUuid }: VisitOverviewComponentPro
           )}
         </TabList>
         <TabPanels>
+          <TabPanel>
+            {isLoadingCurrentVisit ? (
+              <InlineLoading description={`${t('loading', 'Loading')} ...`} role="progressbar" />
+            ) : isError ? (
+              <ErrorState headerTitle={t('activeVisit', 'Active visit')} error={isError} />
+            ) : currentVisit ? (
+              <div className={styles.container}>
+                <div className={styles.header}>
+                  <div className={styles.visitInfo}>
+                    <div>
+                      <h4 className={styles.visitType}>{currentVisit.visitType?.display}</h4>
+                      <div className={styles.displayFlex}>
+                        <h6 className={styles.dateLabel}>{t('start', 'Start')}:</h6>
+                        <span className={styles.date}>{formatDatetime(parseDate(currentVisit.startDatetime))}</span>
+                        {currentVisit.stopDatetime ? (
+                          <>
+                            <h6 className={styles.dateLabel}>{t('end', 'End')}:</h6>
+                            <span className={styles.date}>{formatDatetime(parseDate(currentVisit.stopDatetime))}</span>
+                          </>
+                        ) : null}
+                      </div>
+                    </div>
+                    {visits?.length && <CurrentVisitActions visit={currentVisit} />}
+                  </div>
+                </div>
+                <VisitSummary visit={currentVisit} patientUuid={patientUuid} />
+              </div>
+            ) : (
+              <EmptyState headerTitle={t('visit', 'visit')} displayText={t('Visit', 'Visit')} />
```

Comment:
1. If it's not current visit, it will show EmptyState in the tile?
2. I guess the simple check here should be something like:
```
{visit?.uuid === currentVisit.uuid && <CurrentVisitActions visit={currentVisit} />}
```

Reply:
Come to think of it thats handled on line 57, If there is no current visit, it will show EmptyState